packages/components/src/VirtualizedList/RowLarge/RowLarge.component.js

```diff
@@ -37,6 +37,7 @@ function RowLarge({ className, index, key, parent, style }) {
 			className={classNames(rowThemes)}
 			key={key}
 			style={style}
+			onClick={onRowClick}
```

Comment:
This is not ok. onRowClick should be `onRowClick(event, payload)`, with item as payload.
Change it and align table display with that please